python/requirements.txt

```diff
@@ -32,7 +32,7 @@ flask
 gym
 lz4
 opencv-python-headless==4.3.0.36
-pandas==1.0.5
+pandas >= 1.0.5
```

Comment:
Should be fine, setup.py doesn't put any constraints on pandas versioning and it looks like when the 1.0.5 constraint was put in it was because that was the latest version at the time.